🐛 [Bug] Docker container build fails on Ubuntu 18.04

Hi, I'd like to try the Jupyter notebook here: https://github.com/NVIDIA/TRTorch/tree/master/notebooks 
When I try ```docker build -t trtorch -f notebooks/Dockerfile.notebook .``` I get this error on Step 14:

```
Step 14/21 : RUN bazel build //:libtrtorch --compilation_mode opt
 ---> Running in d67d7bd7b5b8
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Analyzing: target //:libtrtorch (1 packages loaded, 0 targets configured)
Analyzing: target //:libtrtorch (36 packages loaded, 139 targets configured)
INFO: Analyzed target //:libtrtorch (39 packages loaded, 2499 targets configured).
INFO: Found 1 target...
[0 / 76] [Prepa] Writing file include.args
[454 / 520] Compiling core/conversion/evaluators/aten.cpp; 14s processwrapper-sandbox ... (4 actions, 3 running)
[464 / 520] Compiling core/conversion/converters/impl/stack.cpp; 19s processwrapper-sandbox ... (4 actions, 3 running)
ERROR: /workspace/TRTorch/core/lowering/BUILD:10:11: C++ compilation of rule '//core/lowering:lowering' failed (Exit 1) gcc failed: error executing command /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-pa
rameter -Wno-free-nonheap-object -fno-omit-frame-pointer -g0 -O2 '-D_FORTIFY_SOURCE=1' -DNDEBUG -ffunction-sections ... (remaining 61 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
core/lowering/lowering.cpp:2:10: fatal error: torch/csrc/jit/passes/create_functional_graphs.h: No such file or directory
 #include "torch/csrc/jit/passes/create_functional_graphs.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Target //:libtrtorch failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 270.936s, Critical Path: 32.27s
INFO: 19 processes: 19 processwrapper-sandbox.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
```

I'm on a Google Cloud instance with Ubuntu 18.04, Python 3.6 and CUDA 10.2.
Thanks

--- a/.gitignore
+++ b/.gitignore
@@ -37,3 +37,4 @@ bdist
 py/trtorch/_version.py
 py/wheelhouse
 py/.eggs
+notebooks/.ipynb_checkpoints/

--- a/notebooks/Dockerfile.notebook
+++ b/notebooks/Dockerfile.notebook
@@ -7,8 +7,8 @@ RUN apt update && apt install bazel-3.4.1
 RUN ln -s /usr/bin/bazel-3.4.1 /usr/bin/bazel
 
 RUN pip install pillow==4.3.0
-RUN pip install torch==1.5.1
-RUN pip install torchvision==0.6.1
+RUN pip install torch==1.6.0
+RUN pip install torchvision==0.7.0
 RUN pip install notebook
 
 COPY . /workspace/TRTorch

--- a/notebooks/WORKSPACE.notebook
+++ b/notebooks/WORKSPACE.notebook
@@ -53,16 +53,16 @@ http_archive(
     name = "libtorch",
     build_file = "@//third_party/libtorch:BUILD",
     strip_prefix = "libtorch",
-    urls = ["https://download.pytorch.org/libtorch/cu102/libtorch-cxx11-abi-shared-with-deps-1.5.1.zip"],
-    sha256 = "cf0691493d05062fe3239cf76773bae4c5124f4b039050dbdd291c652af3ab2a"
+    urls = ["https://download.pytorch.org/libtorch/cu102/libtorch-cxx11-abi-shared-with-deps-1.6.0.zip"],
+    sha256 = "fded948bd2dbee625cee33ebbd4843a69496729389e0200a90fbb667cdaeeb69"
 )
 
 http_archive(
     name = "libtorch_pre_cxx11_abi",
     build_file = "@//third_party/libtorch:BUILD",
     strip_prefix = "libtorch",
-    sha256 = "818977576572eadaf62c80434a25afe44dbaa32ebda3a0919e389dcbe74f8656",
-    urls = ["https://download.pytorch.org/libtorch/cu102/libtorch-shared-with-deps-1.5.1.zip"],
+    sha256 = "141bb229f4bbf905541096cf8705785e7b0c79e37ca1e5db9d372730b1b9abd7",
+    urls = ["https://download.pytorch.org/libtorch/cu102/libtorch-shared-with-deps-1.6.0.zip"],
 )
 
 ####################################################################################

